react-client-app/src/components/DatasetTbl.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Flex, Spinner, Table, Thead, Tbody, Tr, Th, Td, Spacer, TableCaption, Link, Button, HStack, Select, Text } from "@chakra-ui/react"
+import { useGet, usePatch, chunkIds } from '../api/values'
+import PageHeading from './PageHeading'
+import ConceptTag from './ConceptTag'
+import moment from 'moment';
+
+const DatasetTbl = (props) => {
+    const [datasets, setDatasets] = useState({})
+    const [loadingMessage, setLoadingMessage] = useState("Loading Datasets")
+    const [datapartnerFilter, setDataPartnerFilter] = useState("All");
+    const [title, setTitle] = useState("Datasets");
+
+    useEffect(async () => {
+        // run on initial page load
+        props.setTitle(null);
+        // get datasets and sort by id
+        let datasets = await useGet(`/datasets/`);
+        datasets = datasets.sort((b, a) => (a.id > b.id) ? 1 : ((b.id > a.id) ? -1 : 0));
```

Comment:
I wonder if it's better to sort by creation date, recent to older, instead. I feel like users will want to see more recent entries first

Reply:
This is sorting by id, descending. The recently uploaded datasets are on top of the page so it does the same thing as sorting by creation date would

Reply:
Makes sense